src/drivers/amd/rembrandt/interrupt.c

```diff
@@ -17,7 +17,6 @@
 #include <xtensa/hal.h>
```

Comment:
``Signed-off-by: DINESHKUMAR <DineshKumar.Kalva@amd.com>```

Please write your name as 'FirstName LastName' here.

Reply:
done

Reply:
> `Signed-off-by: DINESHKUMAR <DineshKumar.Kalva@amd.com>`
> 
> Please write your name as 'FirstName LastName' here.

Signed-off-by: DineshKumar Kalva <DineshKumar.Kalva@amd.com> might be the correct sign-off.